### ISSUE TITLE
very nice - future ?

### ISSUE THREAD

**Issue by ghost** [go-ndn/ndn#5]
I was wondering if this has been abandoned.
This is a very nice architectural pattern and implementation.
I am thinking of extending it a bit.

Is the project dead or just your busy on other stuff.
Are you looking for others to help ?

**Comment by taylorchu** [comment]
It is not abandoned. We are simply working on applications that run on top of NDN.

We always need more help. Feel free to open PR.

**Comment by ghost** [comment]
Great to hear. Thanks for the reply.

I will play around with deployment scenarios.
If you have a schematic roadmap of ideas it would be useful to see your
thoughts

**Comment by zjkmxy** [comment]
Hello.
I'm also curious about the status of this project. Is it still being maintained? We changed the packet format a lot in the recent 2 years (removing `Selectors` from and adding `ApplicationParameters` to Interest)
I have read your *The design of go-ndn*. I also used runtime reflection and coroutine to get free from callbacks in my [python-ndn](https://github.com/zjkmxy/python-ndn).

**Comment by taylorchu** [comment]
@zjkmxy no, it is not active in 2020. I might be able to update the packet format and later, but I am waiting for it to stabilize.

In 2018 and 2019, we decided to start with our own tlv format, with a custom implementation, which differs a lot from ndn-cxx. It is also under a different name.

**Comment by zjkmxy** [comment]
Thank you for your reply. Currently no one is using go in our group. I will contribute to this package if there is a need.
Again, your ideas are really nice!
